README.md

```diff
@@ -276,6 +276,8 @@ For information about the governance of the Node.js project, see
 * [codebytere](https://github.com/codebytere) -
 **Shelley Vohr** &lt;codebytere@gmail.com&gt; (she/her)
 * [danbev](https://github.com/danbev) -
+**Danielle Adams** &lt;adamzdanielle@gmail.com&gt; (she/her)
+* [danielleadams](https://github.com/danielleadams) -
```

Comment:
This should be the first line and your name/email the second line.

Reply:
(You'll also want to move these lines below Dan Bevenius's name on the next line and above `davisjam`.